### PR TITLE
10823 update slack installation doc

### PR DESCRIPTION
**Pull request by sanchariGr**
**Proposed changes**:
- As a fix for this [Issue](https://github.com/RasaHQ/rasa/issues/10823#event-6075321987), there is a small update that was needed to the docs.

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Review by ancalita — COMMENTED** [review]
Looks great 🙌🏼 Could you also add a `doc` changelog entry, guidelines are here: https://github.com/RasaHQ/rasa/blob/main/changelog/README.md

**Comment by sanchariGr** [comment]
> Looks great 🙌🏼 Could you also add a `doc` changelog entry, guidelines are here: https://github.com/RasaHQ/rasa/blob/main/changelog/README.md

sorry missed that, on it 👍

File: changelog/10940.doc.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Added an additional step to `Receiving Messages` section in slack.mdx documentation. After a slack update this 
+addiotional step is needed to allow direct messages to the bot.
```
Comment (melindaloubser1) [inline changelog/10940.doc.md:2]:
```suggestion
additional step is needed to allow direct messages to the bot.
```

File: docs/docs/connectors/slack.mdx [hunk 2]
```diff
@@ -110,7 +111,15 @@ your bot and tell you about new messages. If you are running locally, you can
...
-2. Configure the webhook by heading to **Event Subscriptions** and
+2. To send messages directly to your bot using the slack UI, head to **App Home**,
+   scroll to bottom and turn on checkbox for
+   `Allow users to send Slash commands and messages from the messages tab.`
+
+   You might have to quit the Slack app and re-open it before this takes affect
+
+   <Image img={appHomeImg} caption="Messages Tab" alt="Allow users to send Slash commands and messages from the messages tab" max-width="500px"/>
+
+3. Configure the webhook by heading to **Event Subscriptions** and
```
Comment (melindaloubser1) [inline docs/docs/connectors/slack.mdx:122]:
```suggestion
2. To send messages directly to your bot using the slack UI, head to **App Home**,
   scroll to the bottom and select the checkbox for
   `Allow users to send Slash commands and messages from the messages tab.`

   You might have to quit the Slack app and re-open it before your changes take effect.

   <Image img={appHomeImg} caption="Messages Tab" alt="Allow users to send Slash commands and messages from the messages tab" max-width="500px"/>

3. Configure the webhook by heading to **Event Subscriptions** and
```

**Review by melindaloubser1 — CHANGES_REQUESTED** [review]
From the issue, do we need to add more permissions to the end of the section too?

**Comment by sanchariGr** [comment]
> From the issue, do we need to add more permissions to the end of the section too?

@melindaloubser1 I tried both with and without the added permissions mentioned in the issue and it worked both times so i believe whatever was causing this was fixed by slack themselves, and so we dont need to add any other permissions.

**Review by melindaloubser1 — APPROVED** [review]

**Comment by m-vdb** [comment]
woot woot first PR on Open Source @sanchariGr 🎉

**Pull request merged**